### PR TITLE
Add some help if you're on the wrong repo

### PR DESCRIPTION
**Pull request by JeffreyPerplex**
I always do this wrong and never know which repo to switch to

### Prerequisites

- [ ] I have added steps to test this contribution in the description below

If there's an existing issue for this PR then this fixes

### CODE REVIEW

File: .github/BUILD.md [hunk 1]
```diff
@@ -39,6 +39,8 @@ To build Umbraco, fire up PowerShell and move to Umbraco's repository root (the
 
     build/build.ps1
     
+If you only see a build.bat-file, you're probably on the wrong repository. If you switch to the correct repository (dev-v8) the file will appear and you can build it.
```
Comment (nul800sebastiaan) [inline .github/BUILD.md]: If you change "wrong repository" to "wrong branch" then I'll be happy with this change!

Comment (JeffreyPerplex) [reply .github/BUILD.md]: Fixed it... I guess :)

**Comment by nul800sebastiaan** [comment]
Thanks @JeffreyPerplex! 👍

**Comment by JeffreyPerplex** [comment]
By the way: It was this issue: https://github.com/umbraco/Umbraco-CMS/issues/7093

**Pull request merged**